modules/apps/feature-flag/feature-flag-web/src/main/java/com/liferay/feature/flag/web/internal/configuration/admin/category/FeatureFlagConfigurationCategory.java

```diff
@@ -54,7 +55,15 @@ protected void activate(BundleContext bundleContext) {
 			_serviceRegistrations.add(
 				bundleContext.registerService(
 					ConfigurationScreen.class,
-					new FeatureFlagConfigurationScreen(
+					new CompanyFeatureFlagConfigurationScreen(
+						_featureFlagManager, featureFlagType,
+						_featureFlagsDisplayContextFactory, _servletContext),
+					new HashMapDictionary<>()));
+
+			_serviceRegistrations.add(
+				bundleContext.registerService(
+					ConfigurationScreen.class,
+					new SystemFeatureFlagConfigurationScreen(
```

Comment:
No need for separate types on this one. Just pass in the scope as another parameter.